experiments/export_experiment_datasets.py

```
@@ -0,0 +1,241 @@
+            p.categories LIKE '%cs.%' OR 
+            p.categories LIKE '%math.%' OR
+            p.categories LIKE '%stat.%'
+          )
+        ORDER BY v.created_date, p.id
+        LIMIT 75000
+    ),
+    graphsage_papers AS (
+        SELECT p.id, v.created_date
+        FROM arxiv_papers p
+        JOIN arxiv_versions v ON p.id = v.paper_id
+        WHERE v.created_date >= '2012-12-01' 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)